cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/service/http/HttpHandlerGeneratorTest.java

```diff
@@ -261,33 +309,119 @@ protected FileUploadHandler createHandler() {
       HttpURLConnection urlConn = (HttpURLConnection) new URL(
         String.format("http://%s:%d/content/upload/test.txt",
                       bindAddress.getHostName(), bindAddress.getPort())).openConnection();
+      try {
+        urlConn.setReadTimeout(2000);
+        urlConn.setDoOutput(true);
+        urlConn.setRequestMethod("PUT");
+        // Set to use default chunk size
+        urlConn.setChunkedStreamingMode(-1);
+
+        // Write over 150MB of data
```

Comment:
does it have to be that large? this will write 400MB of data to temp files... Isn't 20MB enough to test chunking?